_posts/2020-10-06-solidity-dynamic-array-cleanup-bug.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+published: true
+title:  "Solidity Dynamic Array Cleanup Bug"
+date:   2020-04-06
```

Comment:
I don't think it will happen today :)

Reply:
Funnily that date is not even today, but half a year ago :-D.

Reply:
Ah, copy paste issues.